test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -728,20 +728,21 @@ public class ModelBindingTest
 
             //Assert
             var json = JsonConvert.DeserializeObject<Dictionary<string, string>>(response);
+
             // 8 is the value of MaxModelValidationErrors for the application being tested.
             Assert.Equal(8, json.Count);
-            Assert.Equal("The Field1 field is required.", json["Field1.Field1"]);
-            Assert.Equal("The Field2 field is required.", json["Field1.Field2"]);
-            Assert.Equal("The Field3 field is required.", json["Field1.Field3"]);
-            Assert.Equal("The Field1 field is required.", json["Field2.Field1"]);
-            Assert.Equal("The Field2 field is required.", json["Field2.Field2"]);
-            Assert.Equal("The Field3 field is required.", json["Field2.Field3"]);
-            Assert.Equal("The Field1 field is required.", json["Field3.Field1"]);
+            Assert.Equal("The Field1 field is required.", json["Field1"]);
+            Assert.Equal("The Field2 field is required.", json["Field2"]);
+            Assert.Equal("The Field3 field is required.", json["Field3"]);
+            Assert.Equal("The Field4 field is required.", json["Field4"]);
+            Assert.Equal("The Field5 field is required.", json["Field5"]);
+            Assert.Equal("The Field6 field is required.", json["Field6"]);
+            Assert.Equal("The Field7 field is required.", json["Field7"]);
```

Comment:
Is this the result of the check removed in https://github.com/aspnet/Mvc/pull/2188/files#diff-62a5557f611407fe0ee3bd9e9e2a1393L96 ?